test/test_sox_effects.py

```diff
@@ -11,7 +11,7 @@
 class Test_SoxEffectsChain(common_utils.TorchaudioTestCase):
     backend = 'sox'
 
-    test_filepath = common_utils.get_asset_path("steam-train-whistle-daniel_simon.mp3")
+    test_filepath = common_utils.get_asset_path("steam-train-whistle-daniel_simon.wav")
```

Comment:
In this test suite, there are couple of usage of `torchaudio.load`, but now the test file is `wav` format, can you replace them with `common_utils.load_wav`??